### PR TITLE
EYQG 21 to 31

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Enda Yohannes, further manuscripts.

### CODE REVIEW

File: EndaYohannesQollaGable/EYQG031.xml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                                                <origin>
+                                  <origDate notBefore="1920" notAfter="1970">20th century (probably the beginning or around the middle).</origDate>
+                                                    The manuscript was donated by wayzaro Yālāssabu (baptismal name Walatta Giyorgis)  
+                                                    to the tabot of of Yoḥannǝs Māy Kʷākʷ, of Dabra Wayāfat.
```
Comment (eu-genia) [inline EndaYohannesQollaGable/EYQG031.xml:100]: tābot
of of -> of

File: EndaYohannesQollaGable/EYQG026.xml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                                                <origin>
+                                  <origDate notBefore="1900" notAfter="1980">20th century, probably the beginning or the middle.</origDate>
+                                                    Bought by wayzāro Walata Krǝstos for 29 Ethiopian bǝrr and donated by her to the church of Yoḥannǝs of
```
Comment (eu-genia) [inline EndaYohannesQollaGable/EYQG026.xml:100]: Walatta ?

File: EndaYohannesQollaGable/EYQG021.xml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                                                <origin>
+                                  <origDate notBefore="1850" notAfter="1949">Mid-19th/mid-20th century.</origDate>
+                                  Poorly written.</origin>
```
Comment (eu-genia) [inline EndaYohannesQollaGable/EYQG021.xml:86]: does `poorly written` belong to the `origin`?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: EndaYohannesQollaGable/EYQG031.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG031.xml:100]: ok

File: EndaYohannesQollaGable/EYQG026.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG026.xml:100]: ok

File: EndaYohannesQollaGable/EYQG021.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG021.xml:86]: Changed.

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**